ClassicAssist/UI/ViewModels/ObjectInspectorViewModel.cs

```diff
@@ -235,7 +235,12 @@ private static int CountProperty( int serial, int cliloc, int argumentIndex )
                 return 0;
             }
 
-            return properties.Where( p => p.Cliloc == cliloc ).Select( p => int.Parse( p.Arguments[argumentIndex] ) )
+            return properties.Where( p => p.Cliloc == cliloc ).Select( p => 
+                {
+                if ( int.TryParse( p.Arguments[argumentIndex], out int result ) )
+                    return result;
+                return -1;
```

Comment:
This could be confusing depending on what it looks like in the object viewer, was just the typical fail response I was going with. Since this is an int return through it might make sense to just make it 0.